tests/helpers.py

```diff
@@ -14,3 +16,15 @@ def json(self):
 
 def make_test_response(response_class):
     return type("TestResponse", (response_class, JsonResponseMixin), {})
+
+
+@pytest.fixture
+def docstring_parser_not_installed(monkeypatch):
+    original_imports = builtins.__import__
+
+    def mocked_imports(name, *args, **kwargs):
+        if name == "docstring_parser":
+            raise ImportError()
+        return original_imports(name, *args, **kwargs)
+
+    monkeypatch.setattr(builtins, "__import__", mocked_imports)
```

Comment:
The above change will also allow you to set `parser` as None, and be able to test the non imported behavior.

Reply:
fixed, I removed this helper and just used `patch` to modify `parser` for the test.